tools/clean-package-json.mjs

```diff
@@ -0,0 +1,19 @@
+// Script run during packaging (see package.json) to tidy up the package.json
+// that is included with the released version.
+
+import { readFileSync, writeFileSync } from "node:fs";
+
+const packageJsonPath = "./package.json";
+const packageJsonString = readFileSync(packageJsonPath, "utf8");
+const packageJson = JSON.parse(packageJsonString);
+
+delete packageJson.browserslist;
+delete packageJson.devDependencies;
+delete packageJson.scripts;
+
+// console.log(packageJson);
```

Comment:
❓Do we want to keep the commented out debug logging?

Reply:
@FarrisPandell deleted.